operations/finance.md

```diff
@@ -429,6 +429,103 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `SettlementId` | string | optional | Identifier of the settlement. |
 | `SettledUtc` | string | optional | Settlement date and time in UTC timezone in ISO 8601 format. |
 
+## Update accounting items
```

Comment:
Ordering here does not correspond to ordering in the overview of endpoints.

Reply:
Fixed in another PR 😉